### PR TITLE
updating thandle path

### PR DESCRIPTION
**Pull request by jebrando**

### CODE REVIEW

**Review by anporumb — APPROVED** [review]
:shipit:

File: tests/sf_service_config_ut/sf_service_config_ut_helpers.h [hunk 1]
```diff
@@ -14,7 +14,7 @@
 #include "macro_utils/macro_utils.h"
 
 #include "c_util/rc_string.h"
-#include "c_util/thandle.h"
+#include "c_pal/thandle.h"
```
Comment (dcristoloveanu) [inline tests/sf_service_config_ut/sf_service_config_ut_helpers.h:17]: >c_pal/thandle [](http://example.com/codeflow?start=10&length=13)

How come no deps changes in this PR?
I'd expect the deps to be updates together with switching the repo where we're getting thandle from...

**Review by dcristoloveanu — CHANGES_REQUESTED** [review]
:clock1:

File: tests/sf_service_config_ut/sf_service_config_ut_helpers.h (hunk 1 above)
Comment (jebrando) [reply tests/sf_service_config_ut/sf_service_config_ut_helpers.h:17]: The deps change came in the PR before.  This was a miss.

**Pull request merged**